Cargo.toml

```diff
@@ -27,6 +27,10 @@ members = [
   "ion-hash"
 ]
 
+[features]
+default = []
+ion-hash = ["digest"]
```

Comment:
Does this mean that the optional `digest` dependency should be included when the `ion-hash` feature is enabled? If so, should `sha2` also be in this list?

Reply:
This means that `ion-hash` will automatically bring in `digest`. I don't think it should bring in `sha2` because `sha2` is not required. Someone might want to use md5 or another hash function instead.